ironfish/src/wallet/account/encoder/bech32.ts

```diff
@@ -0,0 +1,114 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { PUBLIC_ADDRESS_LENGTH } from '@ironfish/rust-nodejs'
+import bufio from 'bufio'
+import { Bech32m } from '../../../utils'
+import {
+  AccountImport,
+  KEY_LENGTH,
+  VERSION_LENGTH,
+  VIEW_KEY_LENGTH,
+} from '../../walletdb/accountValue'
+import { AccountEncoder } from './encoder'
+
+export const BECH32_ACCOUNT_PREFIX = 'ironfishaccount00000'
```

Comment:
If this is the new bech32 version, how about a smaller prefix? Bech32 also says a smaller prefix increases checksum quality.

Reply:
Seems good to me, how about "ifaccount"?